.travis.yml

```diff
@@ -127,6 +127,6 @@ script:
       cp ../.coveragerc .;
       COVER_ARGS="--with-coverage --cover-package dipy";
       fi
-    - nosetests --with-doctest --verbose $COVER_ARGS dipy
+    - pytest --doctest-modules --durations=10 -svv $COVER_ARGS dipy
```

Comment:
So, following [here](https://docs.pytest.org/en/latest/usage.html#cmdline), you just need to add `--pyargs` option. 
Here, the new line : 

` pytest --doctest-modules --durations=10 -svv --pyargs dipy`